### DISCUSSION TITLE
Which Library to Use?

### DISCUSSION THREAD

**Discussion by r1g023** [chingu-voyages/v52-tier3-team-40#22] [Q&A] [answered]
Should we go with React, or would Vue be a better fit for our app? Would Vue’s features offer any advantages for our project? Open to thoughts and recommendations on which library might be the best choice.

**Comment by Valeriusdev** [comment, marked as answer]
After some discussion, we chose React for this project! The large community backing React gives us access to plenty of tutorials and tools, and its flexibility makes it easy to integrate with other technologies. Vue is awesome, but React offers advantages that suit our app better.
